gym_sts/spaces/constants/relics.py

```diff
@@ -757,4 +757,5 @@ def ids(self) -> list:
 LOG_NUM_RELICS = math.ceil(math.log(NUM_RELICS, 2))
 
 # Counter values never need to be greater than 10
+MAX_COUNTER = 15
 LOG_MAX_COUNTER = 4
```

Comment:
Can you make this a function of `MAX_COUNTER`?

Reply:
Done. `LOG_MAX_COUNTER = math.floor(math.log(MAX_COUNTER, 2))`